docs/configuration.md

```diff
@@ -366,7 +369,7 @@ expressApp.post('/interaction/:grant/login', async (req, res) => {
 
 // with koa
 router.post('/interaction/:grant/login', async (ctx, next) => {
-  await provider.setProviderSession(ctx.req, ctx.res, { account: 'accountId' });
+  await provider.setProviderSession(req, res, { account: 'accountId' });
```

Comment:
ctx.req, ctx.res got changed to req, res

Reply:
Ah, good catch.  I did a copy and paste from the Express example.

Reply:
Updated